### ISSUE TITLE
[Vets-Website/CI] Provide larger EC2 instance that can be used for pipeline performance testing purposes 

### ISSUE THREAD

**Issue by ddzz** [department-of-veterans-affairs/va.gov-team#36972]
## Description
I'd like to create a test `c5.2xlarge` instance in `vets-website` to test if the larger instance type affects CI performance.

## Background/context 
Currently `vets-website` uses `c5.xlarge` instances.

---
## Tasks
- [x] Create one `c5.2xlarge` EC2 instance that can be used in `vets-website` CI.

## Acceptance Criteria 
- [x] `vets-website` CI can be modified to use a `c5.2xlarge` instance.

---

**Comment by jhouse-solvd** [comment]
@ddzz - We'll discuss with this team to evaluate for this sprint. This should be straightforward. We'll reach out if there are any questions. We will follow up soon.

**Comment by jhouse-solvd** [comment]
@ddzz - Based on other priorities, we may need to wait on this until the next sprint, or pick this up later in the current sprint. 

Would this present any major blockers for your team?

**Comment by ddzz** [comment]
> @ddzz - Based on other priorities, we may need to wait on this until the next sprint, or pick this up later in the current sprint.
> 
> Would this present any major blockers for your team?
> 
> cc: @oseasmoran73

@jhouse-solvd @oseasmoran73 Thanks for the update. No rush on this since it's not a priority or a blocker right now.

**Comment by ddzz** [comment]
@jhouse-solvd have you been able to prioritize this task?

**Comment by jhouse-solvd** [comment]
Hey @ddzz - Thanks for the nudge! I'll check w/ the team and see if we can provide an ETA in the near future.

**Comment by ddzz** [comment]
@jhouse-solvd yes

**Comment by ph-One** [comment]
It appears that all instances are c5.4xlarge.

![image.png](https://images.zenhubusercontent.com/622146ffd0422cdb4d9fdd52/5ba71c1d-b773-4339-8963-606a752b942e)

**Comment by ddzz** [comment]
> It appears that all instances are c5.4xlarge.
> 
> ![image.png](https://camo.githubusercontent.com/3a5d36aab5e2331de8442392304c96153c65023c663423e4aa2f48cb87c7a442/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3632323134366666643034323263646234643966646435322f35626137316331642d623737332d343333392d383936332d363036613735326239343265)

@ph-One thanks for looking into this. Can we create one c5.9xlarge for testing?

**Comment by ddzz** [comment]
> Spoke with @ddzz on the side in terms of how long he will need. Expected testing is a day - week.
> 
> Jeremy pointed out something [interesting](https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/dsva-vagov-utility/gha-runners.tf?rgh-link-date=2022-04-11T18%3A39%3A41Z#L15-L38). `content-build` uses 6 threads, where as `vets-website` uses 4. @ddzz I am going to bump threads after daily deploy. Can you do your testing and see if you are getting the output you expecting?
> 
> If not, we can attempt to bump `instance type` and retest to see if it increases CI performance

I was hoping to create only one c5.9xlarge with the same number of threads for testing. Do all instances need to be updated to c5.9xlarge?

**Comment by oseasmoran73** [comment]
No, but if something happens to that instance (whether it goes unhealthy or reboots), it may cycle and revert to what the AutoScalaing Policy is, the `5.4`. And this is something we would need to keep an eye on. By changing them all to `5.9` for a set time, it ensure that no funkiness can occur should they cycle through

**Comment by oseasmoran73** [comment]
I bumped the threads. Give it a go and see if it helps with CI. If no significant improvement, I will bump instances to next larger one

**Comment by ddzz** [comment]
> I bumped the threads. Give it a go @ddzz and see if it helps with CI. If no significant improvement, I will bump instances to next larger one

@oseasmoran73 I'm not seeing any difference with the new thread count. Can we go up to c5.9xlarge?

**Comment by oseasmoran73** [comment]
@ddzz Yep! Will get that going towards EOD, after deploy goes out

**Comment by oseasmoran73** [comment]
@ddzz PR has been merged. Give it like 15 mins for the changes to kick in. Once you are finished testing, let me know if you need a revert to `5.4`

**Comment by oseasmoran73** [comment]
Important thing. Make sure you are using `self-hosted` and not the dynamic one (how content-build is using it)

**Comment by ddzz** [comment]
@oseasmoran73 We can revert this back to `c5.4xlarge`. I did some testing, and unfortunately there's no performance improvement here.

**Comment by oseasmoran73** [comment]
Instance revert. Closing out ticket. Feel free to spin up a new ticket or reopen if there is anything else on our end we can help out with Darius
